src/Symfony/Component/Config/Tests/Builder/Fixtures/ScalarNormalizedTypes/Symfony/Config/ScalarNormalizedTypes/ListObjectConfig.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Symfony\Config\ScalarNormalizedTypes;
+
+
```

Comment:
:) (same after `use`)

Reply:
@nicolas-grekas This file is auto-generated, I've adjusted the template https://github.com/symfony/symfony/pull/46328/commits/66a896916496037f350bcb550670d79e184d9592